public/stylesheets/register-new-user.css

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
Kinda weird to modify the body style in the styles of the individual components, might want to make a more global css file for defining styles that are beyond individual components.